packages/modules/auth/src/services/auth-module.ts

```diff
@@ -149,6 +149,35 @@ export default class AuthModuleService
     >(providerIdentities)
   }
 
+  updateProviderIdentites(
```

Comment:
This method is auto generated. When we don't overwrite it with some custom logic and just call the service, we don't need to add them here.
And I believe there is actually a typo here, it should be `updateProviderIdentities`

Reply:
Oh gosh, I missed this in the review. Nice catch